src/game/shared/swarm/asw_gamerules.h

```diff
@@ -571,6 +574,9 @@ class CAlienSwarm : public CSingleplayRules
 #ifndef CLIENT_DLL
 	IASW_Map_Builder *m_pMapBuilder;
 	STEAM_CALLBACK( CAlienSwarm, OnSteamRelayNetworkStatusChanged, SteamRelayNetworkStatus_t );
+
+	KeyValues* playersMuted;
```

Comment:
Use hungarian notation (m_pPlayersMuted) so it's clearer that this isn't a local variable